techHttp.js

```diff
@@ -41,73 +41,74 @@ module.exports = function(mockRequest) {
         });
     }
 
-    /**
-     * Simpler form of get
-     *
-     * @param options.url
-     * @param [options.category]
-     * @param [options.timeout]
-     * @param [options.auth]
-     * @param [options.headers]
-     *
-     * Options object is also passed as such to the 'request' library'
-     */
-    function getFromOpts(ruuid, opts) {
-        return wrapRequest(ruuid, "getAsync", _.extend({
-            json: true
-        }, opts), opts.category);
-    }
-
     return {
         on: emitter.on.bind(emitter),
 
-        // TODO [JLE] 'timeout' and 'auth' should be removed and declared instead directly in the given 'options' parameter
-        get: function wrapGet(ruuid, url, category, timeout, auth, options) {
-
-            if (!_.isString(url) && arguments.length === 2) {
-                var opts = url;
-                return getFromOpts(ruuid, opts);
-            } else {
-                return wrapRequest(ruuid, "getAsync", _.extend({
-                    url: url,
-                    json: true,
-                    auth: auth,
-                    timeout: timeout
-                }, options), category);
-            }
+        /**
+         * http get
+         *
+         * @param options.url
+         * @param [options.category]
+         * @param [options.timeout]
+         * @param [options.auth]
+         * @param [options.headers]
+         * @param [options.json]
+         */
+        get: function wrapGet(ruuid, options) {
```

Comment:
If we could maintain the compatibility with a 'single' (excluding 'ruuid') url parameter, it would be better... :-)

Since the URL is mandatory, we could probably rewrite the function into `get: function wrapGet(ruuid, url, options) {...}` :-)